tests/unit/test_repos_repo.py

```diff
@@ -322,9 +322,13 @@ def test_create_pull_private_required_data(self):
 
     def test_create_pull_private(self):
         """Verify the request for creating a pull request."""
-        data = {"title": "foo", "base": "master", "head": "feature_branch"}
+        data = {"title": "foo", "base": "master", "head": "feature_branch", "draft": True}
```

Comment:
This changes the test case to only test the draft code path. For proper coverage there should be two tests, one that tests the draft code path and one that tests the non-draft code path.